### PR TITLE
remove unsigned and signed on Ptr

### PR DESCRIPTION
**Pull request by KristofferC**
These where added in Julia in 1.4.0.

### CODE REVIEW

**Comment by aminya** [comment]
Thanks for the pull request.

File: src/xmlutils.jl [hunk 1]
```diff
@@ -13,8 +13,6 @@ import EzXML: Document, Node
 #     end
 #     print(io, "$(prefix)(<$desc>)")
 # end
-Base.unsigned(x::Ptr) = UInt(x)
-Base.signed(x::Ptr) = Int(x)
```
Comment (aminya) [inline src/xmlutils.jl:17]: Is this required To keep backward compatibility? or the error was only in Julia 1.4?
```
if VERSION < v"1.4.0"
   Base.unsigned(x::Ptr) = UInt(x)
   Base.signed(x::Ptr) = Int(x)
end
```

Comment (KristofferC) [reply src/xmlutils.jl:17]: I'm pretty sure it doesn't exist in 1.3.

Comment (aminya) [reply src/xmlutils.jl:17]: OK, Thanks! I will merge it.

**Pull request merged**